cmd/nerdctl/restart.go

```diff
@@ -35,13 +40,43 @@ func newRestartCommand() *cobra.Command {
 }
 
 func restartAction(cmd *cobra.Command, args []string) error {
-	err := stopAction(cmd, args)
-	if err != nil {
-		return err
+	// Time to wait after sending a SIGTERM and before sending a SIGKILL.
+	var timeout *time.Duration
+	if cmd.Flags().Changed("time") {
+		timeValue, err := cmd.Flags().GetInt("time")
+		if err != nil {
+			return err
+		}
+		t := time.Duration(timeValue) * time.Second
+		timeout = &t
 	}
-	err = startAction(cmd, args)
+
+	client, ctx, cancel, err := newClient(cmd)
 	if err != nil {
 		return err
 	}
+	defer cancel()
+
+	walker := &containerwalker.ContainerWalker{
+		Client: client,
+		OnFound: func(ctx context.Context, found containerwalker.Found) error {
+			if err := stopContainer(ctx, found.Container, timeout); err != nil {
+				return err
+			}
+			if err := startContainer(ctx, found.Container, false); err != nil {
+				return err
+			}
+			_, err = fmt.Fprintf(cmd.OutOrStdout(), "%s\n", found.Req)
```

Comment:
what about print container ID? I think there cound be multiple container of from same `found.Req`

Reply:
see https://github.com/containerd/nerdctl/pull/1223/files#diff-5575f185a28781b4e743991089f023ba8d2a6b6385cf3fdfb77c38ba346806a4R63-R65

Reply:
Still we should print the full ID

Reply:
> Still we should print the full ID

Why not keep the same as docker?

Reply:
hmm Docker seems wrong, but fine to follow the current Docker's behavior